packages/solidity-contracts/contracts/messaging/gateway/FuelERC20Gateway/FuelERC20GatewayV4.sol

```diff
@@ -449,5 +449,5 @@ contract FuelERC20GatewayV4 is
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[49] private __gap;
+    uint256[48] private __gap;
```

Comment:
this is a precautionary update since I noticed while upgrading we didn't got errors
also just wanted to check why are we reducing it by 1 as we added 5 mappings from the prev version